crates/gosub_pipeline/README.md

```
@@ -0,0 +1,119 @@
+
+The painting generates commands that are needed to render pixels onto the tiles. However, it does not execute this painting. It merely generates
+the commands.
+
+The rastering phase will get the tiles and the paint commands and execute the painting per tile into textures.
+
+The final step is compositing. Here we combine the visible tiles in the layers onto the screen. When we have CSS animations like transitions, we
+do not need to repaint the tiles, but merely update the position of the tiles (or their opacity). The compositing will take care of this and returns 
+fully rendered frame.
+
+
+## Passing of data
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)